charts/flink-job/values.yaml

```
@@ -0,0 +1,376 @@
+# Default values for flink-job.
+
+nameOverride: ""
+fullnameOverride: ""
+
+# -- Only used to decrease duplicate configuration of this chart, if image-automation is used as a sub chart.
+# Overrides the local values if given
+global:
+  image:
+    repository: ""
+  imagePullSecrets: []
+
+image:
+  # -- Which image repository to use
+  repository: flink
+  # -- Which image tag to use
+  tag: main
+  # -- Which image sha to use. If used, the `image.tag` is ignored
+  sha: ""
+  # -- Which image pull policy to use
+  pullPolicy: Always
+
+# -- Image pull secrets. A list of `name: <secret-name>`
+imagePullSecrets: []
+
+# -- Which Flink version to use
+version: v1_15
```

```suggestion
version: v1_16
```
I think we should default to 1.16 since this is the version all of our jobs are running currently